docker/docker-compose-production.yml

```diff
@@ -2,245 +2,170 @@ version: '2'
-      - REDIS_LOCATION_THUMBNAIL=redis://se-redis:6379/2
-      - REDIS_LOCATION_USER_AGENT=redis://se-redis:6379/3
-      - FTP_USER=$FTP_USER
-      - FTP_PASS=$FTP_PASS
-      - FTP_IP=$FTP_IP
-      - SECRET_KEY=$SECRET_KEY
-      - YANDEX_SHOP_PASS=$YANDEX_SHOP_PASS
-      - EMAIL_HOST_PASSWORD=$EMAIL_HOST_PASSWORD
-      - ENV_TYPE=$ENV_TYPE
       - PYTHONUNBUFFERED=0
     depends_on:
-      - se-postgres
-      - se-rabbitmq
-      - se-redis
-      - se-source
+      - postgres
+      - rabbitmq
+      - redis
     ports:
       - $VIRTUAL_HOST_PORT
     volumes:
-      - $DEPS_DIR # contains refarm-site modules
-    volumes_from:
-      - se-source
+      # contains refarm-site modules
+      - $DEPS_DIR
+      # source code volume
+      - $SRC_DIR
+      # contains media files
+      - /opt/media/shopelectro/:$SRC_DIR/media/
     networks:
       - se-backend
       - se-frontend
     command: gunicorn shopelectro.wsgi:application -c /etc/gunicorn.py -b 0.0.0.0:$VIRTUAL_HOST_PORT
 
-  se-python-stage:
+  app-stage:
     image: fidals/se:prod
     container_name: se-python-stage
     restart: always
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
+      - env_files/ports
     environment:
       - DJANGO_SETTINGS_MODULE=shopelectro.settings.dev
-      - DJANGO_LOG_LEVEL=$DJANGO_LOG_LEVEL
-      - DATABASE_URL=postgres://$DB_USER:$DB_PASS@se-postgres/$DB_DEV_NAME
-      - REDIS_PASSWORD=$REDIS_PASSWORD
-      - REDIS_LOCATION_DEFAULT=redis://se-redis:6379/0
-      - REDIS_LOCATION_SESSION=redis://se-redis:6379/1
-      - REDIS_LOCATION_THUMBNAIL=redis://se-redis:6379/2
-      - REDIS_LOCATION_USER_AGENT=redis://se-redis:6379/3
-      - FTP_USER=$FTP_USER
-      - FTP_PASS=$FTP_PASS
-      - FTP_IP=$FTP_IP
-      - SECRET_KEY=$SECRET_KEY
-      - YANDEX_SHOP_PASS=$YANDEX_SHOP_PASS
-      - EMAIL_HOST_PASSWORD=$EMAIL_HOST_PASSWORD
+      - POSTGRES_DB=$STAGE_DB_NAME
+      - REDIS_LOCATION_DEFAULT=$REDIS_STAGE_LOCATION_DEFAULT
+      - REDIS_LOCATION_SESSION=$REDIS_STAGE_LOCATION_SESSION
+      - REDIS_LOCATION_THUMBNAIL=$REDIS_STAGE_LOCATION_THUMBNAIL
+      - REDIS_LOCATION_USER_AGENT=$REDIS_STAGE_LOCATION_USER_AGENT
+      - SECRET_KEY=$STAGE_SECRET_KEY
     depends_on:
-      - se-postgres
-      - se-rabbitmq
+      - postgres
+      - rabbitmq
     ports:
       - '$VIRTUAL_HOST_STAGE_EXPOSE_PORT:$VIRTUAL_HOST_STAGE_PORT'
     volumes:
-      - $DEPS_DIR # contains refarm-site modules
-    volumes_from:
-      - se-source
+      # contains refarm-site modules
+      - $DEPS_DIR
     networks:
       - se-backend
       - se-frontend
     command: gunicorn shopelectro.wsgi:application -c /etc/gunicorn.py -b 0.0.0.0:$VIRTUAL_HOST_STAGE_PORT
 
-  se-celery-beat:
+  celery-beat:
     image: fidals/se:prod
     container_name: se-celery-beat
     restart: always
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
     environment:
```

Comment:
@artemiy312 , `environment` key contain no values, docker-compose throws error because of it